src/components/MusicItem/MusicItem.scss

```diff
@@ -0,0 +1,6 @@
+@import "../../components/Variables.scss";
+
+.music-list {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
the list needs to be responsive. use grid in a way that u have responsive web page.

Reply:
Done.